### DISCUSSION TITLE
Do repository forks appear on google search?

### DISCUSSION THREAD

**Discussion by rain1** [community/community#40391] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi!

I have looked through some repositories on github. It seems like their forks do not show up in google search if I search by username and repository name combination. Does github hide forks from google? Any official information on it?

**Comment by jfullstackdev** [comment]
good question, I'm trying it too but seems it's not

so, maybe you can have your own standalone copy as long as you follow the open source license of that

I tried Linux, other than the most famous mirror from Linus Torvalds, there are searches appearing standalone mirrors from other developers, so I think that's the only way
